tests/e2e-kind.sh

```diff
@@ -74,18 +74,15 @@ install_tiller() {
     echo
 }
 
-install_hostpath-provisioner() {
-    # kind doesn't support Dynamic PVC provisioning yet, this is one way to get it working
-    # https://github.com/rimusz/charts/tree/master/stable/hostpath-provisioner
+install_local-path-provisioner() {
+    # kind doesn't support Dynamic PVC provisioning yet, this is one ways to get it working
```

Comment:
The original `way` was correct, grammatically.

Reply:
😄